__tests__/app.test.js

```diff
@@ -26,17 +27,68 @@ describe("GET /api/categories", () =>{
                 expect(category).toHaveProperty("description", expect.any(String))
             })
         })
-    }) //End of 1st test
- 
+    }) 
     test("404 status and message '404 not found' when pass an incorrect URL", () =>{
         return request(app)
         .get("/api/categorie")
         .expect(404)
         .then(({body})=>{
-            console.log(body)
             expect(body.msg).toBe("404 not found")
         })
-    }) //end of 2nd test
-  
+    })
+})
+describe("GET /api/reviews", () =>{
+    test("Should have a length of 4 and 2 properties 'slug' and 'description'", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+            const { reviews } = body
+            expect(reviews).toHaveLength(13)
+            
+            reviews.forEach(category => {
+                expect(category).toHaveProperty("comment_count", expect.any(String))
+                expect(category).toHaveProperty("owner", expect.any(String))
+                expect(category).toHaveProperty("title", expect.any(String))
+                expect(category).toHaveProperty("review_id", expect.any(Number))
+                expect(category).toHaveProperty("category", expect.any(String))
+                expect(category).toHaveProperty("review_img_url", expect.any(String))
+                expect(category).toHaveProperty("created_at", expect.any(String))
+                expect(category).toHaveProperty("votes", expect.any(Number))
+                expect(category).toHaveProperty("designer", expect.any(String))
+            })
+        })
+    })
+    test("Should return the array in desending order", () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+            const { reviews } = body
+            const constclonedReviews = [...reviews]
+            const sortedByReview_id = constclonedReviews.sort((sortA, sortB) => {
+                return sortB.created_at - sortA.created_at
+            })                 
+            expect(reviews[0].created_at).toBe(sortedByReview_id[0].created_at)
+            expect(reviews[12].created_at).toBe(sortedByReview_id[12].created_at)
+        })
```

Comment:
This is expecting the reviews to be in the same order before and after they're sorted. Have a look at the jest-sorted library and see if you can implement that here instead.